lib/vwo/services/operand_evaluator.rb

```diff
@@ -109,6 +109,14 @@ def evaluate_operand?(operand, custom_variables)
         # Call the self method corresponding to operand_type to evaluate the result
         public_send("#{operand_type}?", operand_value, custom_variables_value)
       end
+
+      def evaluate_user?(operand, custom_variables)
+        users = operand.split(',')
+        users.each do |user|
+          return true if user.strip == custom_variables['vwo_user_id']
```

Comment:
strip suppport? 1.9.3?

Reply:
yeah , test cases passing so i guess yes, this one was covered in segment evaluator for white listing.

Reply:
updated: travis passed for all version , so i guess strip is good.